src/test/unit/views/iac-daily-list.test.ts

```diff
@@ -46,11 +45,6 @@ describe('IAC daily cause list page', () => {
         expect(summary[0].innerHTML).contains('Important information', 'Could not find the display summary heading');
     });
 
```

Comment:
A new test should be added in here for the hearingType

Reply:
Added new test for hearingType. Thanks Chris